### PR TITLE
iPad Article Layout

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by coreyfloyd** [comment]
@joewalsh putting it through its paces… so far looking really nice

**Comment by joewalsh** [comment]
Found a bug with highlighting the footer sections... looking into a fix

**Comment by coreyfloyd** [comment]
@joewalsh is the spacing of the button bar items right?

They look a little more spread out in the mocks

**Comment by coreyfloyd** [comment]
(we should probably spread them out a little further even so)

**Comment by joewalsh** [comment]
fixed - good catch

File: Wikipedia/Code/WebViewController.m [hunk 1]
```diff
@@ -335,6 +336,23 @@ - (void)minimizeFindInPage {
     }
 }
 
+- (CGFloat)marginWidthForSize:(CGSize)size {
```
Comment (montehurd) [inline Wikipedia/Code/WebViewController.m]: @joewalsh Should this be marginWidthForWidth? Height isn't used and not sure we'd ever need to consider it...

**Pull request merged**